src/rpc/rawtransaction.cpp

```diff
@@ -213,6 +214,142 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue searchrawtransactions(const JSONRPCRequest& request) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 4)
```

Comment:
```suggestion
    if (request.fHelp || request.params.size() < 1 || request.params.size() > 3)
```

Reply:
Should use the new `.Check()` syntax instead